integration-webcert/src/main/java/se/inera/intyg/minaintyg/integration/webcert/converter/data/value/CodeListValueConverter.java

```diff
@@ -40,26 +22,34 @@ public CertificateDataValueType getType() {
   @Override
   public CertificateQuestionValue convertToValue(CertificateDataElement element) {
     final var value = ((CertificateDataValueCodeList) element.getValue()).getList();
-    if (value == null || value.isEmpty()) {
+
+    if (value == null || value.isEmpty()
+        || !(element.getConfig() instanceof final CertificateDataConfigCheckboxMultipleCode config)) {
```

Comment:
If the config does not match, maybe we should instead return a TECHNICAL_ERROR instead of a NOT_PROVIDED response?